make-distribution.sh

```diff
@@ -121,7 +121,7 @@ if [ $(command -v git) ]; then
 fi
 
 
-if [ ! $(command -v "$MVN") ] ; then
+if [ ! "$(command -v "$MVN")" ] ; then
```

Comment:
Just making sure this works as expected with quotes within quotes? Otherwise LGTM